contrib/pg_tde/src/include/pg_tde_event_capture.h

```diff
@@ -12,6 +12,8 @@
 
 typedef struct TdeCreateEvent
 {
+	uint64		tid;			/* transaction id of the last event trigger,
```

Comment:
Shouldn't the type be `FullTransactionid`?

Reply:
Changed, but I don't think it makes the code better.